### PR TITLE
chore(describe): copilot renaming

### PR DESCRIPTION
**Pull request by iamhopaul123**
Copilot renaming for `describe` package.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: internal/pkg/describe/backend_service.go [hunk 1]
```diff
@@ -15,45 +15,45 @@ import (
...
+		opts.svcDescriber = d
 		return nil
 	}
 	return opts, nil
 }
 
-// NewBackendAppDescriberWithResources instantiates a backend application with stack resources.
-func NewBackendAppDescriberWithResources(project, app string) (*BackendAppDescriber, error) {
-	d, err := NewBackendAppDescriber(project, app)
+// NewBackendServiceDescriberWithResources instantiates a backend service with stack resources.
```
Comment (efekarakus) [inline internal/pkg/describe/backend_service.go:54]: nit: i know that it was missing before :P but it just disambiguates  the constructor
```suggestion
// NewBackendServiceDescriberWithResources instantiates a backend service describer with stack resources.
```

File: internal/pkg/describe/backend_service.go [hunk 2]
```diff
@@ -136,45 +136,45 @@ func (d *BackendAppDescriber) Describe() (HumanJSONStringer, error) {
...
+	App              string             `json:"application"`
 	Configurations   configurations     `json:"configurations"`
 	ServiceDiscovery serviceDiscoveries `json:"serviceDiscovery"`
 	Variables        envVars            `json:"variables"`
 	Resources        cfnResources       `json:"resources,omitempty"`
 }
 
-// JSONString returns the stringified BackendApp struct with json format.
-func (w *backendAppDesc) JSONString() (string, error) {
+// JSONString returns the stringified BackendService struct with json format.
```
Comment (efekarakus) [inline internal/pkg/describe/backend_service.go:161]:
```suggestion
// JSONString returns the stringified backendSvcDesc struct with json format.
```

File: internal/pkg/describe/backend_service.go [hunk 3]
```diff
@@ -136,45 +136,45 @@ func (d *BackendAppDescriber) Describe() (HumanJSONStringer, error) {
...
+func (w *backendSvcDesc) JSONString() (string, error) {
 	b, err := json.Marshal(w)
 	if err != nil {
-		return "", fmt.Errorf("marshal applications: %w", err)
+		return "", fmt.Errorf("marshal service: %w", err)
```
Comment (efekarakus) [inline internal/pkg/describe/backend_service.go:165]:
```suggestion
		return "", fmt.Errorf("marshal backend service description: %w", err)
```

File: internal/pkg/describe/lb_web_service.go [hunk 4]
```diff
@@ -31,163 +31,163 @@ const (
...
 }
 
-type appDescriber interface {
+type svcDescriber interface {
 	Params() (map[string]string, error)
 	EnvOutputs() (map[string]string, error)
 	EnvVars() (map[string]string, error)
 	GetServiceArn() (*ecs.ServiceArn, error)
-	AppStackResources() ([]*cloudformation.StackResource, error)
+	ServiceStackResources() ([]*cloudformation.StackResource, error)
```
Comment (efekarakus) [inline internal/pkg/describe/lb_web_service.go:78]: nit: maybe just `StackResources()` ?

File: internal/pkg/describe/lb_web_service.go [hunk 5]
```diff
@@ -31,163 +31,163 @@ const (
...
 }
 
-// HumanJSONStringer contains methods that stringify app info for output.
+// HumanJSONStringer contains methods that stringify service info for output.
```
Comment (efekarakus) [inline internal/pkg/describe/lb_web_service.go:81]: nit
```suggestion
// HumanJSONStringer contains methods to stringify a description into human or machine-readable formats. 
```

File: internal/pkg/describe/lb_web_service.go [hunk 6]
```diff
@@ -331,35 +331,35 @@ func (s serviceDiscoveries) humanString(w io.Writer) {
...
-	Routes           []*WebAppRoute     `json:"routes"`
+	Routes           []*WebServiceRoute `json:"routes"`
 	ServiceDiscovery serviceDiscoveries `json:"serviceDiscovery"`
 	Variables        envVars            `json:"variables"`
 	Resources        cfnResources       `json:"resources,omitempty"`
 }
 
-// JSONString returns the stringified WebApp struct with json format.
-func (w *webAppDesc) JSONString() (string, error) {
+// JSONString returns the stringified WebService struct with json format.
```
Comment (efekarakus) [inline internal/pkg/describe/lb_web_service.go:346]:
```suggestion
// JSONString returns the stringified webSvcDesc struct in json format.
```

File: internal/pkg/describe/lb_web_service.go [hunk 7]
```diff
@@ -331,35 +331,35 @@ func (s serviceDiscoveries) humanString(w io.Writer) {
...
+func (w *webSvcDesc) JSONString() (string, error) {
 	b, err := json.Marshal(w)
 	if err != nil {
-		return "", fmt.Errorf("marshal applications: %w", err)
+		return "", fmt.Errorf("marshal service: %w", err)
```
Comment (efekarakus) [inline internal/pkg/describe/lb_web_service.go:350]:
```suggestion
		return "", fmt.Errorf("marshal web service description: %w", err)
```

File: internal/pkg/describe/app.go [hunk 8]
```diff
@@ -4,168 +4,52 @@
...
+// App contains serialized parameters for an application.
+type App struct {
+	Name string                `json:"name"`
+	URI  string                `json:"uri"`
+	Envs []*config.Environment `json:"environments"`
+	Apps []*config.Service     `json:"applications"`
+}
+
+// JSONString returns the stringified App struct with json format.
+func (p *App) JSONString() (string, error) {
```
[the remark this answers is not in the document]
Comment (iamhopaul123) [reply internal/pkg/describe/app.go:25]: :O

File: internal/pkg/describe/lb_web_service.go (hunk 4 above)
Comment (iamhopaul123) [reply internal/pkg/describe/lb_web_service.go:78]: Problem is we already have `StackResources` in `stack.go`

**Pull request merged**